VirgilCrypto/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>5.1.0</string>
+	<string>$(MARKETING_VERSION)</string>
```

Comment:
[Suggestion] Marketing version seems to be different for each target. But if you specify version in Info.plist, it will be one for all. Maybe it will be better, since all our versions should be sync anyway?

Reply:
This is what Xcode does automatically(